### PR TITLE
adding settings screen

### PR DESCRIPTION
**Pull request by abdulmth**

### CODE REVIEW

**Review by ghost — DISMISSED** [review]
It seems all changes in `BuildInfoActivity.java` and `ConnectToClientActivity.java` and most changes in `AndroidManifest.xml` are unnecessary. They make a review confusing and may cause unnecessary merge conflicts in the future. Also the unneeded imports will cause Warnings in the lint report.
Please revert those changes and use `git add -p` in the future to add only what should be added.

File: server/src/main/res/layout/activity_connect_to_client.xml [hunk 1]
```diff
@@ -10,7 +10,7 @@
         android:id="@+id/textView"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginTop="140dp"
+        android:layout_marginTop="200dp"
```
Comment (ghost) [inline server/src/main/res/layout/activity_connect_to_client.xml]: why did you change the margin? It seems unrelated to the topic of this pull request.
I am worried that if the margin is too big it might cause problems on devices with a smaller screen.

File: server/src/main/res/layout/activity_connect_to_client.xml [hunk 2]
```diff
@@ -35,6 +35,18 @@
...
+        android:layout_width="40dp"
+        android:layout_height="40dp"
+        android:layout_marginTop="8dp"
+        android:layout_marginEnd="8dp"
+        android:text=""
+        android:background="@drawable/ic_settings"
+        app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintTop_toTopOf="parent"
-        app:layout_constraintVertical_bias="0.395" />
+        />
```
Comment (ghost) [inline server/src/main/res/layout/activity_connect_to_client.xml:50]: I, personally, would prefer an item in the options menu over an always visible button for the settings.

File: server/src/main/res/layout/activity_connect_to_client.xml (hunk 1 above)
Comment (abdulmth) [reply server/src/main/res/layout/activity_connect_to_client.xml]: 60dp is the height of the action bar which is removed by applying the noactionbar theme so the view is located exactly where it is meant to be.

**Comment by abdulmth** [comment]
@zerik sure the changes in the manifest are necessary, if you add an activity, which is in our case it should be added to the manifest.

**Comment by ghost** [comment]
@abdulmth sure, the one line for the settings activity is necessary, but the other four are not. (That is why I made the distinction between the android manifest and the other files.)

**Comment by abdulmth** [comment]
@zerik dude, the other changes aren't really changes, they are the same stuff and these are done by automatic formatting.

**Review by schorliee — APPROVED** [review]
LGTM

**Review by steffenachim — APPROVED** [review]

**Review by schorliee — APPROVED** [review]
